sdk/synapse/Azure.Analytics.Synapse.Spark/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-preview.4 (Unreleased)
 
```

Comment:
If you are getting close to releasing you will want to add some notes here.

Reply:
I'm working on that in another PR now.